app/components/auk/accordion-panel.js

```diff
@@ -0,0 +1,19 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { guidFor } from '@ember/object/internals';
+
+/**
+ *
+ * @argument title {String}
+ * @argument isActive {Boolean}: Determines the active (open) state on display. "false" is the default value.
+ */
+export default class AuAccordionPanelItem extends Component {
+  @tracked isActive = this.args.isActive || false;
```

Comment:
Hierdoor zal  de waarde van `isActive` bepaald worden door de waarde van `@isActive` bij initialisatie van de component, maar idealiter blijft de component zich aanpassen wanneer @isActive onderweg verandert. Defaults beter met behulp van een getter implementeren dus:
```
get isActive() {
  return this.args.isActive || false;
}
```

Reply:
Opmerking is niet meer van toepassing, maar er is wel een aanpassing van naam gebeurd op vlak van argument van `isActive` naar `isDefaultActive`.